andela/ah-leagueOfLegends-frontend
# #159965434 Setup react router version 4

**What does this PR do?**
```
   -This PR setups react router version 4 (React-router-dom)
```
*********************
**Description of Task to be completed?**
```
   -Install and configure the React-router-dom router.
   -Create a dummy route for login and the "/" page.
```
*********************
**How should this be manually tested?**
```
  -After cloning repo, open repo with suitable IDE.
  -Run 'run npm install or yarn install' to install all the dependencies.
  -Start app via 'npm start'
  -Click the dummy routes and see what happens.
```
*********************
**What are the relevant pivotal tracker stories?**
[#159965434](https://www.pivotaltracker.com/n/projects/2192365)
